### PR TITLE
Improve the ReadMe to provide more details and useful links

### PR DESCRIPTION
**Pull request by KieranFrance**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,28 @@
-# BuildVu Docker Image
+# BuildVu Docker Image #
 
-## Build image
-```
-cd docker
-docker build -t idrsolutions/buildvu . 
+BuildVu is a PDF to HTML or SVG conversion tool to allow you to display documents natively in the web browser on any smartphone, tablet, PC or Mac. This docker image can be used to containerise BuildVU making it accessible via a REST API which is perfect for could deployments.
```
Comment (daniel-warren) [inline README.md]:
```suggestion
BuildVu is a PDF to HTML or SVG conversion tool to allow you to display documents natively in the web browser on any smartphone, tablet, PC or Mac. This docker image can be used to containerise BuildVu making it accessible via a REST API which is perfect for cloud deployments.
```

File: README.md [hunk 2]
```diff
@@ -1,10 +1,28 @@
...
+
+## Building the Image ##
+
+To build the image from the source use the following steps.
```
Comment (daniel-warren) [inline README.md]:
```suggestion
To build the image from source, use the following steps.
```

File: README.md [hunk 3]
```diff
@@ -1,10 +1,28 @@
...
+
+- Clone the project from [here](https://github.com/idrsolutions/buildvu-docker)
+- Navigate to the project directory in a terminal
+- Run the following command  
+  ```docker build -t idrsolutions/buildvu .```
+
+## Getting Started ##
+
+In order to use the BuildVu Docker image you will need a license to access the BuildVu war file. If you have not got a license yet, you can find a [free trial or contact us here](https://www.idrsolutions.com/buildvu/pricing).
```
Comment (daniel-warren) [inline README.md]:
```suggestion
In order to use the BuildVu Docker image you will need a license to access the BuildVu war file. If you have not got a license yet, you can find our [free trial or contact us here](https://www.idrsolutions.com/buildvu/pricing).
```

File: README.md [hunk 4]
```diff
@@ -1,10 +1,28 @@
...
+
+Once you have access you can get and run the docker image with the following commands.
```
Comment (daniel-warren) [inline README.md]:
```suggestion
Once you have access, you can get and run the docker image with the following commands.
```

**Review by daniel-warren — CHANGES_REQUESTED** [review]
Changes look promising, I made some suggestions on how I think it could be improved.

**Comment by leonatherton** [comment]
What do you think about swapping the _Getting Started_ and _Building the Image_ sections? My thinking is that most people just want to run the image, whereas building it yourself is for more advanced use-cases and is not part of the standard workflow.

Same applies to PRs for the other products.

**Comment by KieranFrance** [comment]
All suggested changes have been implemented and I reordered the sections as I agree with Leons  thoughts on the order (also implemented in the other PRs)

Please update the review if you are happy with the changes

**Review by daniel-warren — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -1,10 +1,28 @@
...
-```
-cd docker
-docker build -t idrsolutions/buildvu . 
+BuildVu is a PDF to HTML or SVG conversion tool to allow you to display documents natively in the web browser on any smartphone, tablet, PC or Mac. This docker image can be used to containerise BuildVu making it accessible via a REST API which is perfect for cloud deployments.
+
+## Getting Started ##
+
+In order to use the BuildVu Docker image you will need a license to access the BuildVu war file. If you have not got a license yet, you can find our [free trial or contact us here](https://www.idrsolutions.com/buildvu/pricing).
+
+Once you have access, you can get and run the docker image with the following commands.
```
Comment (leonatherton) [inline README.md]:
```suggestion
Once you have the BuildVu war file, you can pull and run the docker image with the following commands:
```

File: README.md [hunk 6]
```diff
@@ -1,10 +1,28 @@
...
 
-## Build image
-```
-cd docker
-docker build -t idrsolutions/buildvu . 
+BuildVu is a PDF to HTML or SVG conversion tool to allow you to display documents natively in the web browser on any smartphone, tablet, PC or Mac. This docker image can be used to containerise BuildVu making it accessible via a REST API which is perfect for cloud deployments.
+
+## Getting Started ##
+
+In order to use the BuildVu Docker image you will need a license to access the BuildVu war file. If you have not got a license yet, you can find our [free trial or contact us here](https://www.idrsolutions.com/buildvu/pricing).
```
Comment (leonatherton) [inline README.md]:
```suggestion
In order to use the BuildVu Docker image you will need a license to access the BuildVu war file. If you have not got a license yet, you can [sign up for a free trial](https://www.idrsolutions.com/buildvu/trial-download).
```

File: README.md [hunk 7]
```diff
@@ -1,10 +1,28 @@
...
+- Navigate to the project directory in a terminal
+- Run the following command  
+  ```docker build -t idrsolutions/buildvu .```
+
+## Documentation ## 
 
-## Run container
-Full details on how this Docker Image can be set up and used can be [found here](https://support.idrsolutions.com/buildvu/tutorials/cloud/docker/deploy-buildvu-on-docker).
+[IDRSolutions BuildVu Cloud Support Page](https://support.idrsolutions.com/buildvu/tutorials/cloud/).  
+[IDRSolutions BuildVu Docker Support Page](https://support.idrsolutions.com/buildvu/tutorials/cloud/docker).
```
Comment (leonatherton) [inline README.md]:
```suggestion
[BuildVu Docker Documentation](https://support.idrsolutions.com/buildvu/tutorials/cloud/docker)
[Contact IDRsolutions](https://www.idrsolutions.com/contact-us)
```

File: README.md [hunk 8]
```diff
@@ -1,10 +1,28 @@
...
+- Clone the project from [here](https://github.com/idrsolutions/buildvu-docker)
+- Navigate to the project directory in a terminal
+- Run the following command  
+  ```docker build -t idrsolutions/buildvu .```
+
+## Documentation ## 
 
-## Run container
-Full details on how this Docker Image can be set up and used can be [found here](https://support.idrsolutions.com/buildvu/tutorials/cloud/docker/deploy-buildvu-on-docker).
+[IDRSolutions BuildVu Cloud Support Page](https://support.idrsolutions.com/buildvu/tutorials/cloud/).
```
Comment (leonatherton) [inline README.md]:
```suggestion
[BuildVu Cloud Documentation](https://support.idrsolutions.com/buildvu/tutorials/cloud/)
```

**Review by leonatherton — CHANGES_REQUESTED** [review]

**Comment by KieranFrance** [comment]
The suggested changes have now been implemented

**Comment by KieranFrance** [comment]
The suggested changes have now been implemented

**Review by leonatherton — APPROVED** [review]

**Pull request merged**